service/history/queues/executable.go

```diff
@@ -36,7 +36,6 @@ import (
 
 	"go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/serviceerror"
-
```

Comment:
nit: let's keep those blank lines that separate server repo imports and imports from other repos.